src/components/admin.tsx

```diff
@@ -189,6 +192,7 @@ function Admin({ user }: adminProps) {
       return false;
     });
     for (const postalCode of postalCodeListing) {
+      setAccordionKeys((existingKeys) => [...existingKeys, postalCode]);
```

Comment:
There is a bug in the accordion state after implement the delta field in the postal code. I shifted the setting of the state out of the listener. This will prevent the accordion from reopening after closing (I think Its reopening due to listener receiving the delta updates).